components/MainHeader.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <nav class="bg-white dark:bg-gray-900 fixed w-full z-20 top-0 start-0 border-b border-gray-200 dark:border-gray-600">
+      <div class="max-w-screen-xl flex flex-wrap items-center justify-between mx-auto p-4">
+        <a href="/" class="flex items-center space-x-3 rtl:space-x-reverse">
+          <img src="/images/diploma.jpeg" class="h-8 rounded-full" alt="Academia logo">
```

Comment:
the file name of the image is not correct. It should be `diploma.jpg`

Reply:
even after the change, site persists in the same form as before